### PR TITLE
Support webhooks when creating predictions

### LINKED ISSUE #17: webhook support
I would like to be able to specify a [`webhook` URL](https://replicate.com/docs/reference/http#create-prediction--webhook) and `webhook_events_filter` when creating predictions with the `predict()` or `createPrediction()` methods.

Based on the current API, maybe that could work something like this:

```js
const prediction = await replicate
  .model("owner/model:version")
  .createPrediction(inputs, {webhook, webhook_eventsFilter});

// ☝🏼 Don't actually know if this is the signature we want
// Just want to illustrate that we need to pass more than `inputs`.
```

cc @erbridge @gr2m

### PR DESCRIPTION
**Pull request by erbridge**
Resolves #17

### CODE REVIEW

**Review by 360macky — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -75,6 +75,33 @@ console.log(prediction.status); // "starting"
 From there, you can fetch the current status of the prediction using
 `await prediction.load()` or `await replicate.prediction(prediction.id).load()`.
 
+You can also provide webhook configuration to have Replicate send POST requests
```
Comment (zeke) [inline README.md:78]: Might be good to have a `## Webhooks` heading here so we can deep link to it.

File: README.md [hunk 2]
```diff
@@ -75,6 +75,33 @@ console.log(prediction.status); // "starting"
...
+to your service when certain events occur:
+
+```js
+import replicate from "replicate";
+
+await replicate
+  .model(
+    "stability-ai/stable-diffusion:db21e45d3f7023abc2a46ee38a23973f6dce16bb082a930b0c49861f96d1e5bf"
```
Comment (zeke) [inline README.md:86]: Conjecture warning: This version is SD 2.x, which had artists removed? Or something? So maybe the warhol prompt might not be the most appropriate.

**Review by zeke — COMMENTED** [review]
I can see the thinking behind introducing a `webhook` object for tidiness, but we might be painting ourselves into a corner here, in that we'd be deviating from the OpenAPI specification. If users were to create a prediction using a generated method that maps to  the [create a prediction](https://replicate.com/docs/reference/http#create-prediction) REST API operation, I would expect it to look something more like this:

```js
{
  input: {}
  webhook: `example.com`
  webhookEventsFilter: ['output', 'completed']
}
```

**Review by zeke — APPROVED** [review]
Love the changes. Looks good!

File: lib/Model.js [hunk 3]
```diff
@@ -122,7 +122,7 @@ export default class Model extends ReplicateObject {
     return prediction;
   }
 
-  async createPrediction(input) {
+  async createPrediction(input, { webhook, webhookEventsFilter } = {}) {
```
Comment (zeke) [inline lib/Model.js:125]: Why does it take two objects? Couldn't this just be one object that looks something like {input = {}, webhook, webhookEventsFilter} ?

(Dupe of my comment here: https://github.com/replicate/replicate-js/pull/20#discussion_r1128706554)

Comment (erbridge) [reply lib/Model.js:125]: Going to deal with this in #20.

**Pull request merged**